atomate/vasp/fireworks/core.py

```diff
@@ -131,232 +129,126 @@ def __init__(
 
-        # Copy SCAN outputs with '.relax3' suffix
-        t.append(
-            CopyFilesFromCalcLoc(
-                calc_loc=True,
-                name_append=".relax3",
-                exclude_files=[
-                    "vdw_kernel.bindat",
-                    "FW.json",
-                    "FW--*",
-                    "*.orig",
-                    "*.relax1",
-                    "*.relax2",
-                    "INCAR.temp"
-                ],
+        elif structure:
+            vasp_input_set = vasp_input_set or MPScanRelaxSet(
+                structure, **vasp_input_set_params
             )
-        )
-
-        # Delete the VdW kernel, WAVECAR, custodian.json, and VASP output files
-        # that have been copied to .relax3
-        # Deleting custodian.json is necessary to avoid double-counting
-        # custodian.json and custodian.json.relax3 when the output is parsed
-        t.append(
-            DeleteFiles(
-                files=[
-                    "vdw_kernel.bindat",
-                    "WAVECAR*",  # All WAVECARs,
-                    "custodian.json",
-                    "*CAR",  # All files that end in "CAR"
-                    "EIGENVAL",
-                    "AECCAR?",
-                    "IBZKPT",
-                    "LOCPOT",
-                    "REPORT",
-                    "std_err.txt",
-                    "vasp.out",
-                    "CHG",
-                    "PCDAT",
-                    "vasprun.xml",
-                    "INCAR.temp"
-                ]
+            t.append(
+                WriteVaspFromIOSet(structure=structure, vasp_input_set=vasp_input_set)
             )
-        )
-
-        # Parse the outputs into the database
-        t.append(VaspToDb(db_file=db_file, additional_fields={"task_label": name}))
+            # Update the INCAR for the PBESol GGA preconditioning step
+            pre_opt_settings = {"_set": {"GGA": "Ps",
+                                         "METAGGA": None,
+                                         "EDIFFG": -0.05
+                                         }}
+
+            # Disable vdW for the precondition step
+            if vasp_input_set_params.get("vdw"):
+                pre_opt_settings.update({"_unset": {"LUSE_VDW": True,
+                                                    "BPARAM": 15.7}})
+
+            t.append(ModifyIncar(incar_dictmod=pre_opt_settings))
+        else:
+            raise ValueError("Must specify structure or previous calculation")
 
-        # gzip the output
+        # Run VASP
+        t.append(RunVaspCustodian(vasp_cmd=vasp_cmd, auto_npar=">>auto_npar<<", gzip_output=False))
+        t.append(PassCalcLocs(name=name))
+        # Parse
+        t.append(VaspToDb(db_file=db_file, **vasptodb_kwargs))
+        # Delete the VdW kernel
+        t.append(DeleteFiles(files=["vdw_kernel.bindat"]))
+        # zip the output (don't rely on custodian to do it)
         t.append(GzipDir())
```

Comment:
Can you get rid of this now and revert to `RunVaspCustodian(... gzip_output=True)`?

Reply:
I'd prefer to leave this as is, so that the workflow can be run without Custodian and still behave exactly the same way. To me, it's much easier to understand and work with if custodian ONLY handles error correction while all other tasks (job construction, parsing, zipping) are explicitly defined in atomate. I don't see a reason why removing custodian should cause the output not to be .gzipped, for example.

Reply:
Ok, that makes sense!